package.json

```diff
@@ -15,17 +15,17 @@
     "url": "https://github.com/andrewkeig/express-validation"
   },
   "dependencies": {
+    "@hapi/joi": "^15.1.0",
```

Comment:
Since `@hapi/joi` is a `peerDependencies`, remove it as `dependencies` and return it as `devDependencies`.